benchmark.rb

```diff
@@ -0,0 +1,140 @@
+require 'json'
+require 'lightstep'
+require 'net/http'
+require 'pp'
+require 'uri'
+
+$base_url = "http://localhost:8000"
+
+$test_tracer = LightStep::Tracer.new(
+  component_name: 'lightstep/ruby/example',
+  transport: LightStep::Transport::HTTPJSON.new(
+    host: 'localhost',
+    port: 8000,
+    encryption: LightStep::Transport::HTTPJSON::ENCRYPTION_NONE,
+    access_token: 'none'
+  )
+)
+$noop_tracer = LightStep::Tracer.new(
```

Comment:
As a tangent, I'm curious how much of a No-Op this really is. I know this is how the code used to work (i.e., pass a nil transport), but it makes me worry that the code is doing work to build something and then not transporting the resulting data.